csharp/elasticbeanstalk/elasticbeanstalk-bg-pipeline/src/ElasticbeanstalkBgPipeline/ElasticbeanstalkBgPipelineStack.cs

```diff
@@ -0,0 +1,75 @@
+using Amazon.CDK;
+using Amazon.CDK.AWS.CodePipeline.Actions;
+using Amazon.CDK.AWS.CodePipeline;
+using Amazon.CDK.AWS.CodeCommit;
+using Amazon.CDK.AWS.Lambda;
+using Amazon.CDK.AWS.S3;
+using System.Collections.Generic;
+
+namespace ElasticbeanstalkBgPipeline
+{
+    public class ElasticbeanstalkBgPipelineStack : Stack
+    {
+        public ElasticbeanstalkBgPipelineStack(Construct parent, string id, IStackProps props) : base(parent, id, props)
+        {
+            var blueEnv = this.Node.TryGetContext("blue_env");
+            var greenEnv = this.Node.TryGetContext("green_env");
+            var appName = this.Node.TryGetContext("app_name");
+
+            var bucket = new Bucket(this, "BlueGreenBucket", new BucketProps{
+                // The default removal policy is RETAIN, which means that cdk destroy will not attempt to delete
+                // the new bucket, and it will remain in your account until manually deleted. By setting the policy to 
+                // DESTROY, cdk destroy will attempt to delete the bucket, but will error if the bucket is not empty.
+                RemovalPolicy = RemovalPolicy.DESTROY // NOT recommended for production code
+            });
+
+            var handler = new Function(this, "BlueGreenLambda", new FunctionProps{
+                Runtime = Runtime.PYTHON_3_7,
+                Code = Code.FromAsset("resources"),
+                Handler = "blue_green.lambda_handler",
+                Environment = new Dictionary<string, string> {
+                    {"BUCKET", bucket.BucketName}
```

Comment:
For examples, lets try to use the cleaner dictionary constructor:
```
Environment = new Dictionary<string, string> 
{
     ["BUCKET"] = bucket.BucketName
};

Reply:
I agree that syntax would probably be better for production code, but it's completely unintuitive to people who haven't seen it before.

Reply:
That's a fair point, but it is what we use in our docs and workshop post-update, so its what we are standardizing around